examples/ethernet/ethernet/main/ethernet_main.c

```diff
@@ -176,8 +386,25 @@ void app_main()
     config.phy_get_duplex_mode = phy_tlk110_get_duplex_mode;
     //Only FULLDUPLEX mode support flow ctrl now!
     config.flow_ctrl_enable = true;
-    config.phy_get_partner_pause_enable = phy_tlk110_get_partner_pause_enable;    
+    config.phy_get_partner_pause_enable = phy_tlk110_get_partner_pause_enable;
     config.phy_power_enable = phy_tlk110_power_enable;
+#endif
+#if (CONFIG_PHY_LAN8720)
+    eth_config_t config;
+    //Only FULLDUPLEX mode support flow ctrl now!
+    config.flow_ctrl_enable = true;
+    config.phy_addr = PHY0;
+    config.mac_mode = ETH_MODE_RMII;
+    config.gpio_config = eth_gpio_config_rmii;
+    config.tcpip_input = tcpip_adapter_eth_input;
+    config.phy_init = phy_lan8720_init;
+    config.phy_check_init = phy_lan8720_check_phy_init;
+    config.phy_check_link = phy_lan8720_check_phy_link_status;
+    config.phy_get_speed_mode = phy_lan8720_get_speed_mode;
+    config.phy_get_duplex_mode = phy_lan8720_get_duplex_mode;
+    config.phy_get_partner_pause_enable = phy_lan8720_get_partner_pause_enable;
+    config.phy_power_enable = phy_lan8720_power_enable;
+#endif
```

Comment:
I'm not sure about the naming here either.

It may make more sense, long term, to split these PHY-specific functions into individual source files and have `extern eth_config_t lan8270_eth_config` and `extern eth_config_t tlk110_eth_config` or something like this.

You don't need to change that in this PR unless you want to, it's something we can rework in a follow-up (eventually some of this will probably be moved to library code, as well.)